### PR TITLE
:construction: Handle delete method

### PR DESCRIPTION
**Pull request by TrAyZeN**

### CODE REVIEW

File: ORA.Tracker/Routes/Members.cs [hunk 1]
```diff
@@ -58,6 +58,7 @@ protected override byte[] post(HttpListenerRequest request, HttpListenerResponse
 
             var c = ClusterDatabase.Get(urlParams["id"])
                 ?? throw new HttpListenerException(404, invalidClusterId);
+            // TODO: Check admin ?
```
Comment (adamaq01) [inline ORA.Tracker/Routes/Members.cs]: yes :p

File: ORA.Tracker/Routes/Members.cs [hunk 2]
```diff
@@ -66,5 +67,29 @@ protected override byte[] post(HttpListenerRequest request, HttpListenerResponse
...
+                ?? throw new HttpListenerException(400, missingMemberId);
+
+            if (!TokenManager.Instance.IsValidToken(token))
+                throw new HttpListenerException(400, invalidToken);
+
+            TokenManager.Instance.RefreshToken(token);
+
+            var c = ClusterDatabase.Get(urlParams["id"])
+                ?? throw new HttpListenerException(404, invalidClusterId);
+            // TODO: Check admin ?
```
Comment (adamaq01) [inline ORA.Tracker/Routes/Members.cs]: yes :p

**Review by adamaq01 — DISMISSED** [review]

File: ORA.Tracker/Routes/Members.cs [hunk 3]
```diff
@@ -58,13 +58,38 @@ protected override byte[] post(HttpListenerRequest request, HttpListenerResponse
 
             var c = ClusterDatabase.Get(urlParams["id"])
                 ?? throw new HttpListenerException(404, invalidClusterId);
-            if (TokenManager.Instance.GetIdFromToken(token) != c.owner)
+            string userId = TokenManager.Instance.GetIdFromToken(token);
+            if (userId != c.owner || !c.admins.Contains(userId))
```
Comment (adamaq01) [inline ORA.Tracker/Routes/Members.cs]: ??? logical or should be a logical and here

File: ORA.Tracker/Routes/Members.cs [hunk 4]
```diff
@@ -58,13 +58,38 @@ protected override byte[] post(HttpListenerRequest request, HttpListenerResponse
...
+
+            if (!TokenManager.Instance.IsValidToken(token))
+                throw new HttpListenerException(400, invalidToken);
+
+            TokenManager.Instance.RefreshToken(token);
+
+            var c = ClusterDatabase.Get(urlParams["id"])
+                ?? throw new HttpListenerException(404, invalidClusterId);
+            string userId = TokenManager.Instance.GetIdFromToken(token);
+            if (userId != c.owner || !c.admins.Contains(userId))
```
Comment (adamaq01) [inline ORA.Tracker/Routes/Members.cs]: same here

**Review by adamaq01 — CHANGES_REQUESTED** [review]

File: ORA.Tracker/Routes/Members.cs (hunk 3 above)
Comment (TrAyZeN) [reply ORA.Tracker/Routes/Members.cs]: Yes obviously, my brain lagged :D

**Review by adamaq01 — APPROVED** [review]

**Pull request merged**